hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -639,10 +641,16 @@ public void testListStatusWithIntermediateDir() throws Exception {
       }, 1000, 120000);
     }
 
-    FileStatus[] fileStatuses = fs.listStatus(parent);
+    List<FileStatus> fileStatuses = new ArrayList<>(Arrays.asList(
+        fs.listStatus(parent)));
+    Path trashRoot = new Path(OZONE_URI_DELIMITER, TRASH_PREFIX);
+    fileStatuses.removeIf(f -> trashRoot.equals(f.getPath()));
```

Comment:
Can we not use the ``listStatus`` with ``PathFilter`` option rather than doing this filtering ourselves?
Something like or a better variant as you like
```
    fs.listStatus(parent, path -> path.toString().startsWith("/.Trash"));
```

Reply:
Thanks for the suggestion, updated.